Makefile

```diff
@@ -307,7 +307,7 @@ verify.generators: verify.repo generate verify.diff
 API_DIR ?= api
 
 .PHONY: generate
-generate: generate.gateway-api-urls generate.crd-kustomize generate.k8sio-gomod-replace generate.mocks generate.cli-arguments-docs
+generate: generate.gateway-api-urls generate.crd-kustomize generate.k8sio-gomod-replace generate.mocks generate.cli-arguments-docs generate.deepcopy generate.apitypes-funcs generate.docs generate.lint-fix generate.format
```

Comment:
I think the `generate.crd` is also required, ref: https://github.com/Kong/kong-operator/pull/2348

Reply:
Do you want me to add generate.crds in this PR, or should we merge #2348 separately?  @randmonkey